libs/blocks/tabs/tabs.js

```diff
@@ -295,6 +296,10 @@ const init = (block) => {
         assocTabItem = rootElem.querySelector(`#tab-panel-${id}-${val}`);
       }
       if (assocTabItem) {
+        const tabLabel = tabListItems[val - 1]?.innerText;
```

Comment:
is val always available and is always a number ? The method that gets `val` is called `getStringKeyName`, is the returned value a string ?

Reply:
so I think it's always a string version of a number. example: '1'. It's parsing the value in that table which is always the id and index number separated by a comma. example: 'acrobat, 1'.
Since this is nested in `if (assocTabItem) {`, it means it found the item on 296. So it is very unlikely to not be a number if they reach this code.  But line 300 tests it did not come back undefined.